src/MakePhpCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace WP_CLI\I18n;
+
+use DirectoryIterator;
+use Gettext\Translations;
+use IteratorIterator;
+use SplFileInfo;
+use WP_CLI;
+use WP_CLI\Utils;
+use WP_CLI_Command;
+
+class MakePhpCommand extends WP_CLI_Command {
+	/**
+	 * Create PHP files from PO files.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <source>
+	 * : Path to an existing PO file or a directory containing multiple PO files.
+	 *
+	 * [<destination>]
+	 * : Path to the destination directory for the resulting PHP files. Defaults to the source directory.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Create PHP files for all PO files in the current directory.
+	 *     $ wp i18n make-php .
+	 *
+	 *     # Create a PHP file from a single PO file in a specific directory.
+	 *     $ wp i18n make-php example-plugin-de_DE.po languages
+	 *
+	 * @when before_wp_load
+	 *
+	 * @throws WP_CLI\ExitException
+	 */
+	public function __invoke( $args, $assoc_args ) {
+		$source = realpath( $args[0] );
+		if ( ! $source || ( ! is_file( $source ) && ! is_dir( $source ) ) ) {
+			WP_CLI::error( 'Source file or directory does not exist!' );
```

Comment:
I don't think the exclamation mark is necessary. We don't generally use such elsewhere.

Reply:
Good point. I think I copied this from the other commabds so I‘ll make sure usage is aligned everywhere.